gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/flow/impl/FlowManagerImpl.java

```diff
@@ -210,5 +237,72 @@ private Policy createPolicy(Step step) {
             return null;
         }
     }
+
+    private void addExecutionFlow(ExtensionPoint extensionPoint, Flow flow, List<Policy> executionPolicies) {
+        Set<ExecutionFlow> existingFlows = policies.get(extensionPoint);
+        if (existingFlows == null) {
+            existingFlows = new HashSet<>();
+        }
+        ExecutionFlow executionFlow = new ExecutionFlow(flow, executionPolicies);
+        existingFlows.remove(executionFlow);
```

Comment:
Because existingFlows is a Set, from my point of view the remove call is useless